src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-daemon-list/service-daemon-list.component.spec.ts

```diff
@@ -75,6 +81,8 @@ describe('ServiceDaemonListComponent', () => {
       daemon_name: 'mon.a',
       daemon_type: 'mon',
       version: '15.1.0-1174-g16a11f7',
+      memory_usage: '17.7',
+      cpu_percentage: '3.54%',
```

Comment:
@avanthakkar looks good But it would be great if you attach some screenshot to it.

Reply:
This is just backport of https://github.com/ceph/ceph/pull/45808#issue-1195884746